queries/rst/highlights.scm

```diff
@@ -56,10 +63,32 @@
  (#match?
   @function.builtin
   ; https://docutils.sourceforge.io/docs/ref/rst/roles.html
-  "^:(emphasis|literal|code|math|pep-reference|rfc-reference|strong|subscript|superscript|title-reference|raw):$"))
+  "^:(emphasis|literal|code|math|pep-reference|PEP|rfc-reference|RFC|strong|subscript|sub|superscript|sup|title-reference|title|t|raw):$"))
+
+; Prefix role
+((interpreted_text
+  (role) @_role
+  "interpreted_text" @text.emphasis)
+ (#eq? @_role ":emphasis:"))
+
+((interpreted_text
+  (role) @_role
+  "interpreted_text" @text.strong)
+ (#eq? @_role ":strong:"))
+
+; Sufix role
+((interpreted_text
+  "interpreted_text" @text.emphasis
+  (role) @_role)
+ (#eq? @_role ":emphasis:"))
+
+((interpreted_text
+  "interpreted_text" @text.strong
```

Comment:
Is this really verbatim "interpreted_text"?

Reply:
Yeah, it's an anonymous node